docs/examples/kueue/kueue.md

```diff
@@ -0,0 +1,31 @@
+# Example of running Kueue with knavigator
+
+## Install kueue
+
+Install kueue by following these [instructions](https://kueue.sigs.k8s.io/docs/installation/):
+
+```bash
+VERSION=v0.6.2
```

Comment:
One minor suggestion: maybe we can use `KUEUE_VERSION`, just in case it overwrites any existing environmental variable in the user's setup.

Reply:
addressed, thanks